channels/fast-4.10.yaml

```diff
@@ -0,0 +1,6 @@
+feeder:
+  delay: PT0H
+  filter: 4\.[9\|10]\.[0-9].*
+  name: fast
+name: fast-4.10
+versions:
```

Comment:
you probably want to run `hack/stabilization-changes.py` locally to populate this, so we don't have zounds of parallel bot PRs as all the supported 4.9.z flow in.  See 4db817a08ecd and #1039.